vendor/github.com/moby/buildkit/go.mod

```diff
@@ -71,6 +72,11 @@ require (
 )
 
 replace (
+	// containerd: Forked from 0edc412565dcc6e3d6125ff9e4b009ad4b89c638 (20201117) with:
+	// - `Adjust overlay tests to expect "index=off"`        (#4719, for ease of cherry-picking #5076)
+	// - `overlay: support "userxattr" option (kernel 5.11)` (#5076)
+	// - `docker: avoid concurrent map access panic`         (#4855)
```

Comment:
Is this related to https://github.com/moby/moby/issues/42422 ?

Reply:
no